test/ErrorHandler-test.js

```diff
@@ -1,14 +1,16 @@
 import { describe, it } from 'mocha'
+import '../src/Promise'
```

Comment:
I factored out some commits that would be useful for creed even without cancellation or handles into my [bergus/creed:master](//github.com/bergus/creed) branch (guess I'll make a PR).
In cancellation, an `Action` needs a truthy `.promise` property to be executed, and for the static `silencer` I chose `never()` to be that property (it could have been anything but I wanted a promise type), which created the circular dependency between ErrorHandler -> silenceError -> silencer -> never -> Promise -> ErrorHandler… It's fine as long as `Promise.js` is included first and the global `errorHandler` can be instantiated. I wasn't particularly happy but it worked :-/
I understood your intentions (and a public `.silence()` method might be more performant and expressive than `.catch(()=>{})`) but I wasn't sure whether `silence` might have been called on futures somewhere (and didn't check). In none of the test cases `silenceError` is called on anything else, but that's not a proof. If you can assert it's not used otherwise, I should probably rewrite this indeed.
Btw, error-tracking might need a major rewrite anyway to be ES7-compatible. The event names for unhandled rejections are standardised now by node and HTML5, and ES7 mandates that the leafs of rejected promise chains are reported not the root.